MaxText/input_pipeline/_hf_data_processing.py

```diff
@@ -243,10 +243,7 @@ def make_hf_eval_iterator(
       token=config.hf_access_token,
   )
 
-  if config.eval_steps > 0:
-    eval_generate_padding_example = True
-  else:
-    eval_generate_padding_example = False
+  eval_generate_padding_example = bool(config.eval_steps)
```

Comment:
Can we revert this change ? Its unlikely, but for negative numbers the changed code evaluates to True, whereas the old code to False.

Reply:
Or even better:
```py
eval_generate_padding_example = config.eval_steps > 0
```